data/json/items/chemicals_and_resources.json

```diff
@@ -540,31 +540,73 @@
     "count" : 10
    },
    {
-    "type" : "AMMO",
+    "type" : "COMESTIBLE",
     "id" : "chem_ethanol",
-    "category" : "spare_parts",
-    "price" : 100,
+    "comestible_type" : "DRINK",
+    "use_action" : "ALCOHOL_STRONG",
+    "nutrition" : 0,
+    "quench" : -16,
+    "healthy" : -10,
+    "addiction_type" : "alcohol",
+    "addiction_potential" : 7,
+    "spoils_in" : 0,
+    "flags" : ["EATEN_COLD"],
+    "fun" : 2,
+    "charges": 17,
```

Comment:
The idea was to roughly match the amount of alcohol in one "charge" of 95% ethanol to the amount of alcohol in one "charge" of vodka/whiskey/etc, which are roughly 40% ethanol (and have "charges": 7, unless my memory fails me). The multiples-of-seven thing could be easily kept by simply saying this has double the concentration, rather than double-plus-a-little, I'll make that change today.